Core/Executor/QueryManager.php

```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Kaliop\eZMigrationBundle\Core\Executor;
+
+use eZ\Publish\API\Repository\Values\Content\Search\SearchResult;
+use Kaliop\eZMigrationBundle\API\Event\BeforeStepExecutionEvent;
+use Kaliop\eZMigrationBundle\API\Event\StepExecutedEvent;
+use Kaliop\eZMigrationBundle\API\ExecutorInterface;
+use Kaliop\eZMigrationBundle\API\MatcherInterface;
+use Kaliop\eZMigrationBundle\API\Value\MigrationStep;
+use Kaliop\eZMigrationBundle\Core\Matcher\QueryTypeMatcher;
+use Kaliop\eZMigrationBundle\Core\MigrationService;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\ExpressionLanguage\ExpressionFunction;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+
+class QueryManager extends RepositoryExecutor
```

Comment:
What about `QueryExecutor` instead?
Besides, class name isn't quite self-explanatory. Maybe a phpdoc comment is needed 😉